RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -810,6 +810,21 @@ void finalizeMaximize(boolean speculative)
 			addBonusToMaximize($item[spring shoes], 50);
 		}
 	}
+
+	if(auto_haveBatWings() && get_property("_batWingsFreeFights").to_int() < 5)
+	{
+		if(!nextMonsterIsFree)
+		{
+			addBonusToMaximize($item[bat wings], 200); // get the 5 free fights
+		}
+		else
+		{
+			// bat wings free fights can get swallowed by some free fights
```

Comment:
apparently this was fixed

https://discord.com/channels/466605739838930955/474736566246572042/1292842816728469566

Reply:
oh sweet thanks for the info